libs/common/domain/src/lib/model/search/filter.model.ts

```diff
@@ -2,5 +2,21 @@ import { FieldName } from './field.model'
 
 export type FieldFilterByValues = Record<string, boolean>
 export type FieldFilterByExpression = string | number
-export type FieldFilter = FieldFilterByExpression | FieldFilterByValues
+export type FieldFilterByRange = {
+  start?: Date
```

Comment:
What is the reason behind making `start` and `end` optional?

Reply:
It may be interesting to allow users filtering for all records before one date or after one date in the future.